src/components/inspector/device-visibility.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="form-group">
+    <div>
+      <label>{{ label }}</label>
+
+      <b-form-checkbox
+        v-model="settings.showForDesktop"
+        :value="true"
+        :unchecked-value="false"
+        switch
+      >
+        {{ $t("Show for Desktop") }}
+      </b-form-checkbox>
+
+      <b-form-checkbox
+        v-model="settings.showForMobile"
+        :value="true"
+        :unchecked-value="false"
+        switch
+      >
+        {{ $t("Show for Mobile") }}
+      </b-form-checkbox>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  components: {},
+  props: ["label", "value", "helper", "options"],
+  data() {
+    return {
+      settings: {
+        showForDesktop: true,
+        showForMobile: true
+      }
+    };
+  },
+  watch: {
+    settings: {
```

Comment:
can a computed property be possible here? Trying to avoid watcher

Reply:
oh, you can add a @change property to it. If the v-model is ever changed, that would trigger the response